news/2 Fixes/11811.md

```diff
@@ -0,0 +1 @@
+Changed `telemetry.telemetryEnabled` to `telemetry.telemetryLevel` in `README.md` under "Data and telemetry".
```

Comment:
Please create an issue and then add the issue number here instead of the PR number